airflow/www_rbac/.eslintignore

```diff
@@ -0,0 +1,10 @@
+**/*{.,-}min.js
+**/*.sh
+**/*.py
+gantt-chart-d3v2.js
+jqClock.min.js
+coverage/**
+static/dist/*
+static/docs/*
+node_modules/*
+node_modules*/*
```

Comment:
What's the second node_modules entry for?

Reply:
nice catch @tedmiston . In fact, we don't have to mention the `node_modules` at all here in .eslintignore file. ESLint always ignores files in `/node_modules/*` and `/bower_components/*` by default. Will remove that.